### ISSUE TITLE
How to add uploader?

### ISSUE THREAD

**Issue by Tigzy** [DinoTools/dionaea#101]
Hello, 
How can I add some python script to upload payloads to custom location?

**Comment by Tigzy** [comment]
OK, so I found the submit_http_post, filled the config and created the symlink. 
However it doesn't get called (unlike hpfeeds that I filled the same way), is it a bug?

**Comment by phibos** [comment]
The submit_http_post incident handler does only handle ```dionaea.download.complete.unique``` incidents. This means the payload will only be submitted the first time your honeypot sees it.

**Comment by DZKeeper** [comment]
I am also very interested in this feature. Care to give more details on how to setup this ?

More so, on how to setup this on the dionaea itself and on the receiving server

I have a few dionaea instance, how will dedup happen, any ideas, tips ?

Thanks in advance

**Comment by DZKeeper** [comment]
@Tigzy : were you able to do this with mrf ?

**Comment by Tigzy** [comment]
@DZKeeper yes, here: http://www.adlice.com/catch-malware-with-your-own-honeypot-v2/

**Comment by Tigzy** [comment]
@phibos complete.unique doesn't seem to be called in submit_http_post. probably a bug.

**Comment by JBAnderson** [comment]
Part 4 of [this](http://www.attacusatlas.com/how-to-set-up-dionaea-honeypot-with-modern-honey-network-and-slack-alerts/) blog post is probably what you are looking for. I'm the author so let me know if you run into any trouble
